PowerShell.Common.props

```diff
@@ -1,10 +1,97 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--
+        We should *explicitly* assign version properties in this target
+        because:
+          1. By default the version properties are defined in global 'PropertyGroup' (not target!) in 'Microsoft.NET.DefaultAssemblyInfo.targets'
+          2. Properties in global 'PropertyGroups' are assigned before any targets
+          3. and cannot be conditionally excecuted as targets (after this target).
```

Comment:
> and cannot be conditionally excecuted as targets (after this target).

First, a typo I didn't notice in my last change: excecuted  -> executed.
Second, I have trouble understanding this sentence accurately. Can you please reword this sentence a bit to make it easier to understand?
And one more thing, what does `global PropertyGroup` mean? Is it different from the `PropertyGroup` we used in this file?

Reply:
Typo fixed.
Under "global" I meant that they affect the entire project. Removed.
I tryed to reword but ... - feel free to rewrite.